lib/route_format.rb

```diff
@@ -1,7 +1,7 @@
 module RouteFormat
 
   def self.username
-    /[\w.\-]+?/
+    /[%\w.\-]+?/
```

Comment:
This is because encoding url client side will create this kind of usernames:

```js
encodeURI("東京都") // %E6%9D%B1%E4%BA%AC%E9%83%BD
```

If you don’t you will have this kind of errors:

```
No route matches [GET] "/u/%E6%9D%B1%E4%BA%AC%E9%83%BD/preferences"
```